packages/functional-tests/src/tests/sound-test.ts

```diff
@@ -0,0 +1,102 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as MRESDK from '@microsoft/mixed-reality-extension-sdk';
+import { Actor } from '@microsoft/mixed-reality-extension-sdk';
+import App from '../app';
+import delay from '../utils/delay';
+import destroyActors from '../utils/destroyActors';
+import Test from './test';
+
+export default class SoundTest extends Test {
+
+    constructor(app: App, private baseUrl: string) {
+        super(app);
+    }
+
+    public async run(): Promise<boolean> {
+        let success = true;
+
+        success = success && await this.runSoundTest();
+
+        return success;
+    }
+
+    public async runSoundTest(): Promise<boolean> {
+        // Make a root object.
+        const tester = MRESDK.Actor.CreateEmpty(this.app.context, {});
+
+        const textPromise = Actor.CreateEmpty(this.app.context, {
+            actor: {
+                name: 'label',
+                parentId: tester.value.id,
+                transform: {
+                    position: { x: 0, y: 2, z: 0 }
+                },
+                text: {
+                    anchor: MRESDK.TextAnchorLocation.MiddleCenter,
+                    color: { r: 30 / 255, g: 206 / 255, b: 213 / 255 },
+                    height: 0.3
+                }
+            }
+        });
+
+        const text = textPromise.value;
+        await textPromise;
+        text.text.contents = "Preloading Sound";
+
+        const soundId = this.app.context.assetManager.loadSound(
+            'sounds', `${this.baseUrl}/testsound.wav`,
+            {
+                looping: true
+            });
+
+        await soundId;
```

Comment:
>await soundId; [](start = 8, length = 14)

Can you rewrite this test to not require awaiting soundId? text.playSound should be immediately callable. (actor should await on the sound to load before playing it). This is consistent with the way animations work.